lib/uxtx.js

```diff
@@ -0,0 +1,307 @@
+const moment = require('moment');
+const assert = require('bsert');
+
+const { Amount, TX } = require('bcoin');
+
+// TODO: rename this.getJSON to
+// this.getTXJSON
+// TODO: value to BTC, prepend + or -
+
+
+// bcoin api
+// deposits:
+// null path in the inputs
+// output with path defined and change false
+//
+// withdrawals:
+// input with defined path and change false
+// output with path defined and change true
+
+class TXUX extends TX {
```

Comment:
Just brainstorming some other names for this class:
- UITX 
- WalletTX
- InterfaceTX
- TXInterface
- HumanReadableTX (a little long but get's the point across 🤷‍♂️ )
- ExplorerTX / TXExplorer

Reply:
• DisplayTX
• UserTX
• SmartTX

Reply:
- TXEntended
- CustomTX
- TXCustom

Reply:
My favorite so far is `DisplayTX`. Extended and Custom feel a little too general.